code/objects/items/equippable.dm

```diff
@@ -0,0 +1,25 @@
+/obj/item/equippable
```

Comment:
why did you rename these?
both equipable and equippable are correct[1].

[1] https://en.wiktionary.org/wiki/equipable